test/custom_task_test.go

```diff
@@ -1204,3 +1205,30 @@ func tearDownV1Beta1CustomTask(ctx context.Context, t *testing.T, c *clients, na
 
 	tearDown(ctx, t, c, namespace)
 }
+
+// updateConfigMap updates the config map for specified @name with values. We can't use the one from knativetest because
```

Comment:
why do we need to add this in this PR?

Reply:
It was also deleted from the deleted 'test/artifact_bucket_test' and moved here since it is used in this file.